product_variant_default_code/tests/test_variant_default_code.py

```diff
@@ -11,7 +11,7 @@ class TestVariantDefaultCode(common.SavepointCase):
     @classmethod
     def setUpClass(cls):
         super(TestVariantDefaultCode, cls).setUpClass()
-        cls.sale_setting = cls.env['sale.config.settings'].create({})
+        cls.base_setting = cls.env['base.config.settings'].create({})
```

Comment:
We need to force the group to the user instead or the test is not going to get the config:
```python
cls.group_default_code = cls.env.ref(
            'product_variant_default_code.group_product_default_code')
```

Reply:
done